versions.cfg

```diff
@@ -2,6 +2,15 @@
 # Buildout
 setuptools =
 zc.buildout =
+plone.restapi =
```

Comment:
The empty `plone.restapi` pin should not be needed: this is done in `base.cfg` already.